opentrons-ai-client/src/resources/utils/protocolFormat.ts

```diff
@@ -0,0 +1,32 @@
+import type { Chat, ProtocolFormat } from '../types'
+
+/**
+ * Automatically detects the protocol format based on message content and history
+ * @param message - The current user message
+ * @param history - Optional chat history
+ * @returns ProtocolFormat - 'Protocol Designer' or 'Python'
+ */
+export const detectProtocolFormat = (
+  message: string,
+  history?: Chat[]
+): ProtocolFormat => {
+  // Check current message for Protocol Designer keywords
+  if (message && message.toLowerCase().includes('protocol designer')) {
```

Comment:
this may cause an issue since `message` in opentrons-ai-client/src/molecules/InputPrompt/index.tsx can be '' and toLowerCase gets an issue.

```suggestion
  if (message.length > 0 && message.toLowerCase().includes('protocol designer')) {
```

Reply:
thanks!